### PR TITLE
feat(taskrun): update default tab

### PR DESCRIPTION
**Pull request by abhinandan13jan**
## Fixes 

https://issues.redhat.com/browse/RHTAPBUGS-1068

## Description

Navigayes to details page for succeeded Taskruns, else navigates to log pages.

## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

Screenshot 2024-01-11 at 3 54 13 PM

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/components/TaskRunDetailsView/TaskRunDetailsView.tsx [hunk 1]
```diff
@@ -51,7 +53,14 @@ export const TaskRunDetailsView: React.FC<React.PropsWithChildren<TaskRunDetails
...
   }
 
-  const applicationName = taskRun.metadata?.labels[PipelineRunLabel.APPLICATION];
+  const applicationName = taskRun?.metadata?.labels[PipelineRunLabel.APPLICATION];
+  const baseURL = `/application-pipeline/workspaces/${workspace}/applications/${applicationName}/taskruns/${taskRunName}`;
+  const { activeTab } = params;
+
+  if (!activeTab && trStatus) {
+    trStatus === runStatus.Succeeded ? navigate(`${baseURL}/details`) : navigate(`${baseURL}/logs`);
+  }
```
Comment (sahil143) [inline src/components/TaskRunDetailsView/TaskRunDetailsView.tsx:62]: move the `navigate` call inside a useEffect

**Review by sahil143 — CHANGES_REQUESTED** [review]

File: src/components/TaskRunDetailsView/TaskRunDetailsView.tsx (hunk 1 above)
Comment (abhinandan13jan) [reply src/components/TaskRunDetailsView/TaskRunDetailsView.tsx]: updated

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Comment by rohitkrai03** [comment]
@abhinandan13jan You might need to update e2e tests.

**Pull request merged**